lib/redcord/attribute.rb

```diff
@@ -79,5 +74,19 @@ def should_range_index?(type)
     end
   end
 
+  module InstanceMethods
+    extend T::Sig
+
+    sig { returns(T.nilable(String)) }
+    def hash_tag
+      attr = self.class.class_variable_get(:@@shard_by_attribute)
+
+      return nil if attr.nil?
+
+      # A blank hash tag would cause MOVED error in cluster mode
+      "{#{send(attr) || '__redcord_hash_tag_null__'}}"
```

Comment:
will add a validation here just incase attr happen to be the same as the magic value `__redcord_hash_tag_null__`